docs/hub/models-cards-co2.md

```diff
@@ -20,11 +20,11 @@ You can add the carbon footprint data to the model card metadata (in the README.
 ```yaml
 ---
 co2_eq_emissions:
-      emissions: "in grams of CO2"
-      source: "source of the information, either directly from AutoTrain, code carbon or from a scientific article documenting the model"
-      training_type: "pre-training or fine-tuning"
-      geographical_location: "as granular as possible, for instance Quebec, Canada or Brooklyn, NY, USA"
-      hardware_used: "how much compute and what kind, e.g. 8 v100 GPUs"
+  emissions: "in grams of CO2"
+  source: "source of the information, either directly from AutoTrain, code carbon or from a scientific article documenting the model"
+  training_type: "pre-training or fine-tuning"
+  geographical_location: "as granular as possible, for instance Quebec, Canada or Brooklyn, NY, USA"
```

Comment:
Do we want to give more information about how to find one's electricity grid here? E.g.
"To find your local electricity grid, check out [Electricity Map](https://app.electricitymap.org/)."

This can help people figure out what granularity to use, since there is like 1 grid in all of France, versus dozens in the USA.